src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import ReactDOM from 'react-dom';
+import "./modal.scss";
+
+let modalsCount = 0;
+
+class Modal extends React.Component {
+    constructor( props ) {
+        super( props );
+
+        this.container = document.getElementById('modal-container');
+
+        if (!this.container) {
+            this.container = document.createElement('div');
+            this.container.id = 'modal-container';
+            document.body.appendChild(this.container);
+        }
+    }
+
+    componentDidMount() {
+        modalsCount++;
```

Comment:
i'm quite confused by this counter 🤔

Reply:
fixed, the counter doesn't have sense now